docs/source/guides/integrations.mdx

```diff
@@ -0,0 +1,279 @@
+   (...)
+
+   def _save_pretrained(self, save_directory: Path):
+      """Generate Model Card and save weights from a Pytorch model to a local directory."""
+      model_card = ModelCard.from_template(
+         card_data=ModelCardData(
+            license='mit',
+            library_name="pytorch",
+            ...
+         ),
+         model_summary=...,
+         model_type=...,
+         ...
+      )
+      (save_directory / "README.md").write_text(str(model))
+      torch.save(obj=self.module.state_dict(), f=save_directory / "pytorch_model.bin")
+```
+
+3. Implement the `_from_pretrained` method:
+
+```python
+class PyTorchModelHubMixin(ModelHubMixin):
+   (...)
+
+   @classmethod # Must be a classmethod!
+   def _from_pretrained(
+      cls,
+      *,
+      model_id: str,
+      revision: str,
+      cache_dir: str,
+      force_download: bool,
+      proxies: Optional[Dict],
+      resume_download: bool,
+      local_files_only: bool,
+      token: Union[str, bool, None],
+      map_location: str = "cpu", # additional argument
+      strict: bool = False, # additional argument
+      **model_kwargs,
+   ):
+      """Load Pytorch pretrained weights and return the loaded model."""
+      if os.path.isdir(model_id): # Can either be a local directory
+         print("Loading weights from local directory")
+         model_file = os.path.join(model_id, "pytorch_model.bin")
+      else: # Or a model on the Hub
+         model_file = hf_hub_download( # Download from the hub, passing same input args
+            repo_id=model_id,
+            filename="pytorch_model.bin",
+            revision=revision,
+            cache_dir=cache_dir,
+            force_download=force_download,
+            proxies=proxies,
+            resume_download=resume_download,
+            token=token,
+            local_files_only=local_files_only,
+         )
+
+      # Load model and return - custom logic depending on your framework
+      model = cls(**model_kwargs)
+      state_dict = torch.load(model_file, map_location=torch.device(map_location))
+      model.load_state_dict(state_dict, strict=strict)
+      model.eval()
+      return model
+```
+
+And that's it! Your library now enables users to upload and download files to and from the Hub.
+
+## Quick comparison
+
+Let's quickly sum up the two approaches we saw with their advantages and drawbacks. The table below is only indicative.
+Your framework might have some specificities that you need to address. This guide is only here to give guidelines and
+ideas on how to handle integration. In any case, feel free to contact us if you have any questions!
+
+<!-- Generated using https://www.tablesgenerator.com/markdown_tables -->
+| Integration | Using helpers | Using [`ModelHubMixin`] |
+|:---:|:---:|:---:|
+| User experience | Good.<br>`model = load_from_hub(...)`<br>`push_to_hub(model, ...)` | Good.<br>`model = MyModel.from_pretrained(...)`<br>`model.push_to_hub(...)` |
```

Comment:
Maybe remove "Good" here since they're both good and it doesn't really tell you much; might be better to only compare usage.